e2e/mobile/specs/ledgerSync/ledgerSync.spec.ts

```diff
@@ -30,7 +30,7 @@ async function initializeLedgerKeyRingProtocol() {
   return CLI.ledgerKeyRingProtocol({
     initMemberCredentials: true,
     apiBaseUrl: ledgerKeyRingProtocolArgs.apiBaseUrl,
-  }).then((output: { pubkey: string; privatekey: string }) => {
+  }).then(output => {
     if (output && "pubkey" in output) {
```

Comment:
I was wondering about LedgerSync issue. Maybe we should fail here if the condition is not fulfilled. It means that the command failed. To do in another PR of course.